src/SprykerEco/Zed/Optivo/Dependency/Facade/OptivoToMoneyFacadeInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+namespace SprykerEco\Zed\Optivo\Dependency\Facade;
+
+use Generated\Shared\Transfer\MoneyTransfer;
+
+interface OptivoToMoneyFacadeInterface
+{
+    /**
+     * @param int $value
+     *
+     * @return \Generated\Shared\Transfer\MoneyTransfer
+     */
+    public function fromInteger(int $value): MoneyTransfer;
```

Comment:
Please use exactly the same method typehinting like in MoneyFacadeInterface.

Reply:
sorry, what is wrong here?